### ISSUE TITLE
`,apropos-doc` cannot find functions defined in REPL

### ISSUE THREAD

**Issue by HiPhish** [bakpakin/Fennel#462]
Hello,

the `,apropos-doc` and `,apropos-show-docs` cannot find functions defined on the REPL. Here is what I am doing in a fresh Fennel REPL started from the command-line:

```
$ fennel
Welcome to Fennel 1.3.1 on PUC Lua 5.3!
Use ,help to see available commands.
>> (fn herp [] "Herp derp lol" :herp)
#<function: 0x559fe514bcf0>
>> (fn derp [] "Herp derp rofl" :derp)
#<function: 0x559fe4f8e450>
>> ,apropos-doc derp

>> ,apropos-show-docs derp
```

First I define two functions `herp` and `derp` which have similar (but not same) docstrings. Then I search for the pattern `derp`. The first comma-command prints an empty line, the second one does nothing. However, the `,apropos` command can find them just fine:

```
>> ,apropos erp
___repl___.env.___replLocals___.derp	___repl___.env.___replLocals___.herp
```

**Comment by technomancy** [comment]
Yeah, currently those commands only search loaded modules, but they could certainly be extended to also search repl locals. Would be happy to take a patch for this.
